src/app/(app)/home/entries/[slug]/page.jsx

```diff
@@ -0,0 +1,49 @@
+import { getEntry } from '@/app/actions/entries.js';
+import { getEntries } from '@/app/actions/entries.js';
+import styles from './viewer.module.scss';
+
+export async function generateStaticParams() {
+  // TODO: replace user id
+  const entries = await getEntries(1);
+
+  return entries.map((entry) => ({
+    slug: entry.slug,
+  }));
+}
+
+export default async function Page({ params }) {
+  const entryResponse = await getEntry(params.slug);
+  const entry = entryResponse[0];
+
+  return (
+    <section className={styles.viewerContainer}>
+      <section className={styles.viewerDetails}>
+        <div className={styles.createdTimeDate}>
+          <span className={styles.date}>
```

Comment:
Both of these spans are using the `date` class. Is there a way that you think we can leverage the power of SCSS to reduce this redundancy?

Reply:
funny thing is, no styles are even assigned to the date class haha, just removed it! 
but yes it looks like targeting the children of `.createdTimeDate` would've saved me from writing a class, though I ended up just putting the styles in `.createdTimeDate` itself. if there was something else you were getting at lmk!

Reply:
Nope that seems right to me